lib/db.js

```diff
@@ -0,0 +1,18 @@
+import sqlite from 'sqlite'
+import Promise from 'bluebird'
+import { db } from '../lib'
+
+export async function connectDatabase (dbName) {
+  return sqlite.open(dbName, { Promise })
+}
+
+export async function createTable (dbPath, tableParameters) {
+  const db = await connectDatabase(dbPath)
+  await db.run('create table if not exists ' + tableParameters)
+
+  return db
```

Comment:
We dont need to await twice here. Just do `return db.run('create table if not exists ' + tableParameters)`